runtime/libffi/closures.c

```diff
@@ -492,7 +492,9 @@ dlmmap_locked (void *start, size_t length, int prot, int flags, off_t offset)
 	  close (execfd);
 	  goto retry_open;
 	}
-      ftruncate (execfd, offset);
+      if (ftruncate (execfd, offset)) {
+        /* ignore any failure. */
```

Comment:
Either
```c
        /* ignore any failure */
```
or
```c
        /* Ignore any failure. */
```
If a sentence fragment, the period is not needed, if a full sentence, then it should start with a capital.